pombola/settings/base.py

```diff
@@ -202,6 +204,12 @@
 
 MIDDLEWARE_CLASSES = (
     'django.middleware.gzip.GZipMiddleware', # first in list so it is able to act last on response
+)
+
+if os.environ.get("DJANGO_DEBUG_TOOLBAR", "false").lower() == "true":
+    MIDDLEWARE_CLASSES += ("debug_toolbar.middleware.DebugToolbarMiddleware",)
+
```

Comment:
I moved the debug toolbar's middleware to the here because it [needs](https://django-debug-toolbar.readthedocs.io/en/latest/installation.html#enabling-middleware) to be as close as possible to the top, but after `GZipMiddleware`.